Tasks/KubernetesManifestV0/task.loc.json

```diff
@@ -14,7 +14,7 @@
   "version": {
     "Major": 0,
     "Minor": 154,
```

Comment:
0.155.0

Reply:
There are multiple fixes going in. I'll update the last one with `0.155.0`